README.md

```diff
@@ -564,6 +564,7 @@ Table of Contents
   * [statusgator.com](https://statusgator.com/) — Status page monitoring, 3 monitors free
   * [thousandeyes.com](https://www.thousandeyes.com/) — Network and user experience monitoring. 3 locations and 20 data feeds of major web services free
   * [thundra.io/apm](https://www.thundra.io/apm) — Application monitoring and debugging. Has a free tier up to 250k monthly invocations.
+  * [upptime.js.org](https://upptime.js.org) — GitHub-powered open-source uptime monitor and status page. Free, Open Source Project under MIT License
```

Comment:
Nice and all but this isnt SaaS, this is self hosted sofware in GHA that's at worst unlikely to last long and at best an abuse of GHA's generosity.  

I am not really keen to promote this

Reply:
Fair point, it's almost an add-on to GHA.
I still think that is nice and could be well used by developers, so I am in keen to promote it, I saw this one promoted on HN so it's becoming popular really fast.

Reply:
Github is being super generous by providing GHA for free, abusing it is not going to end well for anyone.  Tools like this is not within the spirit of things with such frequent never ending action runs, its very clearly an abuse of the spirit of things.

So I'll pass on this, appreciate the contribution though.